test/BaselineOutput/SingleDebug/SavePipe/SavePipeIidChangePoint-Schema.txt

```diff
@@ -16,7 +16,7 @@
   fAnomaly: Vec<R4, 4>
     Metadata 'SlotNames': Vec<Text, 4>: Length=4, Count=4
       [0] 'Alert', [1] 'Raw Score', [2] 'P-Value Score', [3] 'Martingale Score'
----- IidChangePointDetector ----
+---- RowToRowMapperTransform ----
```

Comment:
You can move this tests to Common/SavePipe folder, or you need to do same thing in SingleRelease/SavePipe

Reply:
made changes in SingleRelease too

---
In reply to: [225355661](https://github.com/dotnet/machinelearning/pull/1254#discussion_r225355661) [](ancestors = 225355661)